monitor/alerts.js

```diff
@@ -59,6 +58,8 @@ async function main() {
       mostRecentTxHash: xAffirmationsMostRecentTxHash,
       transactions: xAffirmationsTxs
     },
+    homeBlockNumber,
```

Comment:
Consider to publish the info about the block range in `getBalances.json`. It is entry point to the monitoring stats so it will be useful to see the block range here.

Reply:
Agree, done